### DISCUSSION TITLE
[API request] Perhaps simply parse (plaintext) expression into function

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/roslyn#66871] [Ideas]
If I am writing a one liner code that's a function that says ```public static int Add (int a, int b) => a + b;```, maybe let me give this line as a string to an API and I get back a function I can invoke the same way

**Comment by CyrusNajmabadi** [comment]
You can use CallerArgumentExpression for this

**Reply by Xyncgas** [reply]
Thanks for the insight, is this available to use, also does this mean it's going to do a string equality check with the expression I typed in code to find the method I am going to use instead of parsing a line of ```plaintext``` to executable function

**Reply by CyrusNajmabadi** [reply]
Yes.  this is available for use.
